docs/frontend-templates/homepage.md

```diff
@@ -246,7 +269,16 @@ insights:
     date: July 2024
     description:
       # You MUST wrap this in single quotation marks (ie. ' '), since markdown can be used to enter this information. To create a heading, use three hashes (ie. ###).
-      'Information regarding the design update, taken place in November 2023'
+      'Information regarding the design update, taken place in November 2023. 
+      
+      Changes included:
+
+      - a visual refresh (larger font sizes, more spacing between sections, making the design clearer on mobile devices)
+      
+      - changes to the page structure based on what we know about what users do and what they expect (removing unused sections, 
+      - showing list of topics as a list vs grid, adding bullets to the list of popular links)
```

Comment:
Is this supposed to be another list item? Currently not rendered as that.

Reply:
thanks for spotting. That should not be there. I tried to weasle my way through github to amend this but got confused. Not sure if the amend came through.

Reply:
Right on lets work on this together